engine/runtime/src/main/java/org/enso/interpreter/runtime/type/Types.java

```diff
@@ -52,6 +52,7 @@
   EnsoDateTime.class,
   EnsoTimeOfDay.class,
   EnsoTimeZone.class,
+  EnsoDuration.class
```

Comment:
The ultimate goal is to get rid of all `TruffleObject` classes in this `Types` list. Technically nothing happens if we remove all date&time related ones. The important once to remove are however `Array` and `Vector` - tracked as [Array vs. ArrayOverBuffer](https://www.pivotaltracker.com/n/projects/2539304/stories/183058716) and [many builtins](https://www.pivotaltracker.com/n/projects/2539304/stories/183266964).